### PR TITLE
Update steam sdk v1.61

### PR DESCRIPTION
**Pull request by TaiJiYu**
1. update steam sdk to v1.61
```go
// Upgraded SteamAPI_SteamUserStats_v012 to SteamAPI_SteamUserStats_v013 in 1.61
flatAPI_SteamUserStats = "SteamAPI_SteamUserStats_v012" -> "SteamAPI_SteamUserStats_v013"
```
2. Mark RequestCurrentStats as deprecated
In version 1.61, the RequestCurrentStats interface was removed from the steam SDK, as described in the steam changelog
```go
// Deprecated: RequestCurrentStats is no longer necessary and has been removed. The Steam Client will synchronize this data before your game launches.
func (s steamUserStats) RequestCurrentStats() bool
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 *~
 steamworks_sdk_*.zip
-.DS_Store
+.DS_Store
```
Comment (hajimehoshi) [inline .gitignore]: ?

File: go.sum [hunk 2]
```diff
@@ -1,2 +1,2 @@
 golang.org/x/sys v0.20.0 h1:Od9JTbYCk261bKm4M/mw7AklTlFYIa0bIp9BgSm1S8Y=
-golang.org/x/sys v0.20.0/go.mod h1:/VUhepiaJMQUp4+oa/7Zr1D23ma6VTLIYjOOTFZPUcA=
+golang.org/x/sys v0.20.0/go.mod h1:/VUhepiaJMQUp4+oa/7Zr1D23ma6VTLIYjOOTFZPUcA=
```
Comment (hajimehoshi) [inline go.sum]: ?

**Comment by hajimehoshi** [comment]
Please do not add Windows DLLs.

File: README.md [hunk 3]
```diff
@@ -59,4 +59,4 @@ These binary files are copied from Steamworks SDK's `redistribution_bin` directo
 
 ## Resources
 
- * [Steamworks SDK](https://partner.steamgames.com/doc/sdk)
+ * [Steamworks SDK](https://partner.steamgames.com/doc/sdk)
```
Comment (hajimehoshi) [inline README.md]: What's the change?

**Comment by hajimehoshi** [comment]
> Mark RequestCurrentStats as deprecated

What if we call RequestCurrentStats? Does this just crash? If so, we can remove the function itself as this library is still version v0.

File: README.md (hunk 3 above)
Comment (TaiJiYu) [reply README.md]: It didn't change, but these prompts appeared automatically after submission :see_no_evil:

**Comment by TaiJiYu** [comment]
> > Mark RequestCurrentStats as deprecated
> 
> What if we call RequestCurrentStats? Does this just crash? If so, we can remove the function itself as this library is still version v0.
Yes, the following panic will occur
panic: Failed to find SteamAPI_ISteamUserStats_RequestCurrentStats procedure in steam_api64.dll: The specified procedure could not be found

File: steamworks_windows.go [hunk 4]
```diff
@@ -318,15 +318,6 @@ func SteamUserStats() ISteamUserStats {
 
 type steamUserStats uintptr
 
-func (s steamUserStats) RequestCurrentStats() bool {
```
Comment (hajimehoshi) [inline steamworks_windows.go:321]: Remove the same function at steamworks_unix.go

File: README.md (hunk 3 above)
Comment (hajimehoshi) [reply README.md]: OK then revert the change.

File: steamworks_windows.go (hunk 4 above)
Comment (TaiJiYu) [reply steamworks_windows.go:321]: ok

**Comment by hajimehoshi** [comment]
Note to myself: https://steamcommunity.com/groups/steamworks/announcements/detail/4480612432780198328 `RequestCurrentStatus` was actually removed.

File: go.sum (hunk 2 above)
Comment (TaiJiYu) [reply go.sum]: I found the reason, my editor automatically removed the last line break, and now I've added it

File: README.md (hunk 3 above)
Comment (TaiJiYu) [reply README.md]: done

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**